lib/oneaws/client.rb

```diff
@@ -32,13 +33,20 @@ def issue_credential(options)
       mfa = response.mfa
 
       # sent push notification to OneLogin Protect
-      mfa_device = mfa.devices.select{|device| device.type == "OneLogin Protect"}&.first
+      mfa_device = mfa.devices[0]
```

Comment:
細かいけどfirstメソッドを使ったほうがRubyっぽくてよさそう～

```suggestion
      mfa_device = mfa.devices.first
```

Reply:
https://github.com/pepabo/oneaws/pull/4/commits/2e7790420ff68aa4931ca348182d31bc5d00f469
取り込みました！